spec/support/matchers/be_valid_verbose.rb

```diff
@@ -0,0 +1,17 @@
+RSpec::Matchers.define :be_valid_verbose do
+  match do |model|
```

Comment:
I'm not confident of changing this one and it doing the right thing. Can anyone easily explain it, or is it okay as-is?